app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<link href="stylesheets/index.css" rel="stylesheet" type="text/css">
```

Comment:
You shouldn't need to do have this line in order to load your CSS. Rails should know how to load its own CSS